src/video/groupnetcamview.cpp

```diff
@@ -140,22 +140,20 @@ GroupNetCamView::GroupNetCamView(int group, QWidget *parent)
     splitter->addWidget(scrollArea);
     scrollArea->setWidget(widget);
 
-    connect(&Audio::getInstance(), &Audio::groupAudioPlayed, this, &GroupNetCamView::groupAudioPlayed);
-
     QTimer* timer = new QTimer(this);
     timer->setInterval(1000);
-    connect(timer, &QTimer::timeout, this, &GroupNetCamView::findActivePeer);
+    connect(timer, &QTimer::timeout, this, &GroupNetCamView::onUpdateActivePeer);
```

Comment:
Unless otherwise specified (i.e. there is an algorithm that's being devised), the activity algorithm should respond instantly to some form of stimulus and not on 1 second update intervals. Ideally a public thread-safe function is supplied (or a public slot) such that CoreAV or other audio/video processing code can signal activity when appropriate stimulus is given.